README.md

```diff
@@ -37,6 +38,26 @@ Solr.configure do |config|
     # define a dynamic field
     f.dynamic_field :text, solr_name: '*_text'
   end
+
+  # Define multiple cores with fields
+  # Core's URL is 'http://localhost:8983/cars'
+  config.define_core(name: :cars) do |f|
+    f.field :manufacturer
+    f.field :model
+  end
+end
+```
+
+Single core configuration:
```

Comment:
can you please add to readme from simple to complex, starting from env var only and then single core and then multiple cores?

Reply:
will improve README